lib/travis/api/v3/services/beta_migration_requests/create_internal.rb

```diff
@@ -0,0 +1,26 @@
+module Travis::API::V3
+  class Services::BetaMigrationRequests::CreateInternal < Service
```

Comment:
Here again, the nomenclature is a bit misleading - how about `ProxyCreateOnCom`?

Reply:
Naming is always hard 😀